examples/notebooks/simple_ape.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
Why are you adding this notebook?

Reply:
found it useful to use something like this when debugging ape. bots_on_solidity used to take 30 seconds to get to the ape part (it's faster now). but it's still nice to have something that is a simple test of ape. both for tessting, and also to show users a clean ape implementation, if they want to test their own setup, or take it in a different direction than we did with bots_on_solidity.